.gitignore

```diff
@@ -22,3 +22,4 @@ crt_portal/NO_ZIP.txt
 *.mo
 # this is where we store uploaded attachments when USE_LOCALSTACK=False
 attachments/
+README.md
```

Comment:
speaking of README.md, can you add a paragraph on how to use this to README.md?

Reply:
will do!